dvc/repo/live.py

```diff
@@ -10,15 +14,27 @@
     from dvc.path_info import PathInfo
     from dvc.repo import Repo
 
+webbrowser_open = call_only_once(webbrowser.open)
```

Comment:
You can use [`funcy.once`](https://funcy.readthedocs.io/en/stable/flow.html#once) instead. Btw, is it possible to have multiple `live` outputs? How does it work with this?

Reply:
Didn't know about funcy.once, thanks!

>  Btw, is it possible to have multiple live outputs?

It is not possible. A stage can only have 1 `live` output. There might be multiple plots in the `live` output directory but that doesn't affect the number of times the HTML generation is called.

Reply:
But can multiple stages have multiple live outputs?

Reply:
Yes. 

I assume that you are pointing to better use `once_per_args`?